Source/ASVideoNode.mm

```diff
@@ -58,6 +58,7 @@ @interface ASVideoNode ()
     unsigned int delegateVideoNodeDidSetCurrentItem:1;
     unsigned int delegateVideoNodeDidStallAtTimeInterval:1;
     unsigned int delegateVideoNodeDidRecoverFromStall:1;
+    unsigned int delegateVideoNodeDidFailToLoadValuesForKey:1;
```

Comment:
Nit: `delegateVideoNodeDidFailToLoad`**Value**`ForKey`, instead of `delegateVideoNodeDidFailToLoad`**Values**`ForKey`